### ISSUE TITLE
Editor crash on linux when pressing shift-v (drag pivot)

### ISSUE THREAD

**Issue by wivlaro** [godotengine/godot#23180]
**Godot version:**
3.0.6-stable_x11.64

**OS/device including version:**
Linux

**Issue description:**
Editor crash
```
ERROR: operator[]: FATAL: Index p_index=0 out of size (size()=0)
   At: core/vector.h:138.
handle_crash: Program crashed with signal 4
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] /lib/x86_64-linux-gnu/libc.so.6(+0x3ef20) [0x7f7d08267f20] (??:0)
[2] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x188a577] (??:?)
[3] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x130d402] (??:?)
[4] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x117480e] (<artificial>:?)
[5] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x938be5] (??:?)
[6] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x1b4ffc3] (??:?)
[7] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x1b3c9a9] (??:?)
[8] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x1b3d6c9] (??:?)
[9] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x121a283] (<artificial>:?)
[10] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x124f304] (<artificial>:?)
[11] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x1250c28] (??:?)
[12] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x1250d1b] (<artificial>:?)
[13] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x938be5] (??:?)
[14] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x1b4ffc3] (??:?)
[15] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x1ad2c11] (??:?)
[16] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x11d6934] (??:?)
[17] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x11e126f] (??:?)
[18] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x7270d5] (??:?)
[19] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x7dd3a3] (<artificial>:?)
[20] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x6e97cc] (??:?)
[21] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xe7) [0x7f7d0824ab97] (??:0)
[22] /home/bill/software/Godot_v3.0.6-stable_x11.64() [0x6f61ce] (??:?)
-- END OF BACKTRACE --

```

**Steps to reproduce:**
Create Control node in blank scene
Press shift-v to try and set pivot.
It crashes with the above stack trace

**Minimal reproduction project:**

**Comment by groud** [comment]
Note that this is fixed on master

**Comment by akien-mga** [comment]
Closing then as it was fixed in 3.1.
